### DISCUSSION TITLE
Forks

### DISCUSSION THREAD

**Discussion by Apep-the-snek** [jugeeya/UltimateTrainingModpack#274] [Q&A]
are there any good forks of this project?

related rant: i hate how github has this fork feature, but alsmost all forks are just garbage nonsense, would be nice to sort them by some sort of poplarity of some kind.

**Comment by jugeeya** [comment]
The main branch at jugeeya/UltimateTrainingModpack is really the only maintained one, others are primarily other users' forks for creating PRs for new features here.
